contracts/Datastore.sol

```diff
@@ -159,14 +196,26 @@ contract Datastore is AragonApp {
         fileList.setFileName(_fileId, _newName);
     }
 
+    /**
+     * @notice Changes encryption key of file `_fileId` to `_cryptoKey`
+     * @param _fileId File Id
+     * @param _cryptoKey Encryption key    
+     */
+    function setEncryptionKey(uint _fileId, string _cryptoKey) public {
+        require(hasWriteAccess(_fileId, msg.sender));
+
+        fileList.setEncryptionKey(_fileId, _cryptoKey);
+        emit FileContentUpdate(msg.sender);
```

Comment:
Shouldn't this event include the `fileId` too?

Reply:
Same thing as previous comment  :)